### PR TITLE
Fix #637

### LINKED ISSUE #637: Suppress warnings only
### Type of Issue
Feature Request

### Description
Looking for an option that will disable warnings in VSCode, while still displaying errors. Something akin to eslint's quiet [flag](https://eslint.org/docs/user-guide/command-line-interface#handling-warnings).

### PR DESCRIPTION
**Pull request by ilias-t**

### CODE REVIEW

File: server/src/eslintServer.ts [hunk 1]
```diff
@@ -873,7 +874,10 @@ function validate(document: TextDocument, settings: TextDocumentSettings, publis
 		if (report && report.results && Array.isArray(report.results) && report.results.length > 0) {
 			let docReport = report.results[0];
 			if (docReport.messages && Array.isArray(docReport.messages)) {
-				docReport.messages.forEach((problem) => {
+				docReport.messages.filter((problem) => {
```
Comment (dbaeumer) [inline server/src/eslintServer.ts]: Instead of filtering and creating another array can we extend the if(problem) in the for each?

Comment (ilias-t) [reply server/src/eslintServer.ts]: done

**Comment by ilias-t** [comment]
Lmk how the changes look and if there's anything still blocking a merge, thanks

**Comment by dbaeumer** [comment]
Thanks for the nice work.

**Pull request merged**